Add acknowledgements to README (#94)
modelcontextprotocol/csharp-sdk: README.MD

The official C# SDK for the [Model Context Protocol](https://modelcontextprotocol.io/), enabling .NET applications, services, and libraries to implement and interact with MCP clients and servers.

> > [!NOTE]

Author: Sorry about the whitespace changes here -- VSCode automatically strips trailing whitespace.

I can revert these if needed.

Reviewer: The whitespace deletions are fine.

await Task.Delay(Timeout.Infinite);
```

## Acknowledgements

> The starting point for this library was a community project called [mcpdotnet](https://github.com/PederHP/mcpdotnet) developed by [Peder Holdgaard Pederson](@PederHP). We are grateful for the work done by Peder to create a solid foundation for this library.

Reviewer: ```suggestion
The starting point for this library was a project called [mcpdotnet](https://github.com/PederHP/mcpdotnet), initiated by [Peder Holdgaard Pederson](@PederHP). We are grateful for the work done by Peder and other contributors to that repository, which created a solid foundation for this library.
```
